R/R-pkgconfig/Portfile

```diff
@@ -14,5 +14,3 @@ checksums           rmd160  c6360d0c6237ab56c553b3a515ca9bf452557e43 \
                     sha256  c371a82737112adcae905ddec116d1e5f8c779fb802e681cfe1f8c7027bfeacf \
                     size    6482
 supported_archs     noarch
-
-depends_lib-append  port:pkgconfig
```

Comment:
Modifying library dependencies generally warrants a revbump ([see docs](https://guide.macports.org/#reference.keywords)).

Reply:
@harens Thank you!

Reply:
Thanks for point this out. It is now fixed.